code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -434,11 +434,12 @@
 			if(show_message)
 				to_chat(M, "<span class='danger'>You feel your wounds fade away to nothing!</span>" )
 	..()
-
+			
 /datum/reagent/medicine/mine_salve/on_mob_end_metabolize(mob/living/M)
 	if(iscarbon(M))
 		var/mob/living/carbon/N = M
 		N.hal_screwyhud = SCREWYHUD_NONE
+		REMOVE_TRAIT(M, TRAIT_PAINKILLER, "Painkiller:MINERS SALVE") //SKYRAT EDIT, Painkiller.
```

Comment:
Painkiller:MINERS SALVE should be a define

Reply:
on it